service/history/historyReplicator.go

```diff
@@ -868,6 +894,51 @@ func (r *historyReplicator) terminateWorkflow(ctx context.Context, domainID stri
 	})
 }
 
+func (r *historyReplicator) getLatestCheckpoint(replicationInfo1 map[string]*h.ReplicationInfo,
```

Comment:
Can you rename replicationInfo1 and replicationInfo2 to replicationInfoLocal and replicationInfoRemote?

Reply:
sure